src/kili/services/plugins/upload.py

```diff
@@ -118,14 +134,65 @@ def _create_plugin_runner(self):
         result = self.auth.client.execute(GQL_CREATE_PLUGIN_RUNNER, variables)
         return format_result("data", result)
 
+    def _check_plugin_runner_status(self, update=False):
+        """
+        Check the status of a plugin's runner until it is active
+        """
+
+        logger = self.get_logger()
+
+        action = "updated" if update else "created"
+
+        logger.info("Plugin is being %s... This should take approximately 3 minutes.", action)
+
+        n_tries = 0
+        status = None
+
+        logger = self.get_logger("INFO")
+
+        while n_tries < 20:
```

Comment:
Can you create a constant at the top of the file to make that clearer?

Reply:
Yes, updated it